src/styles.scss

```diff
@@ -18,7 +43,145 @@
 }
 
 .iframe-video {
-  border: 0px solid #000000;
-  background: #000000bf;
-  box-shadow: 0 0.25vmax 1vmax #000000bf;
+  aspect-ratio: 16 / 9;
+  width: 100%;
+  border: 0;
+}
+
+// Long description truncate
+.description-truncate {
+  display: -webkit-box;
+  -webkit-line-clamp: 3;
+  -webkit-box-orient: vertical;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  margin-bottom: 1rem;
+  line-height: 1.5;
+  max-height: 6em; /* fallback for non-webkit browsers: 1.5 line-height * 4 lines */
+}
+
+// Live video indicator
+.p-navigation__item--live {
+  margin-left: auto !important;
+}
+
+.p-navigation__item--live .p-navigation__link {
+  display: flex;
+  align-items: center;
+}
+
+.live-dot {
+  display: inline-block;
+  width: 8px;
+  height: 8px;
+  background-color: #c7162b;
+  border-radius: 50%;
+  margin-right: 0.5rem;
+  animation: pulse 2s infinite;
+  flex-shrink: 0;
+}
+
+.p-navigation__item--live .p-navigation__link {
+  display: flex;
+  align-items: center;
+  color: #fff;
+}
+
+.p-navigation__item--live .p-navigation__link:hover {
+  color: #fff;
+  text-decoration: underline;
+}
+
+@keyframes pulse {
+  0% { opacity: 1; }
+  50% { opacity: 0.5; }
+  100% { opacity: 1; }
+}
+
+.video-card {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+
+  .p-card__image {
+    position: relative;
+    width: 100%;
+  }
+
+  .thumbnail-wrapper {
+    position: relative;
+    width: 100%;
+    padding-top: 56.25%; // 16:9 aspect ratio
+    overflow: hidden;
+  }
+
+  .p-card__inner {
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+    padding: 1rem;
+  }
+
+  .video-title {
+    display: -webkit-box;
+    -webkit-line-clamp: 2;
+    -webkit-box-orient: vertical;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    line-height: 1.5;
+    max-height: 3em; /* fallback for non-webkit browsers: 1.5 line-height * 2 lines */
+  }
+
+  .video-presenters {
+    display: -webkit-box;
+    -webkit-line-clamp: 1;
+    -webkit-box-orient: vertical;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    line-height: 1.5;
+  }
+
+  img {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+  }
+
+  .duration-badge {
+    position: absolute;
+    padding: 0.25rem;
+    background: rgba(17, 17, 17, 0.80);
+    color: #fff;
+    border-radius: 2px;
+    font-size: 0.75rem;
+    line-height: 1.5;
+    margin: 0.5rem;
+    z-index: 1;
+    border-radius: 5px;
+  }
+}
+
+// Markdown styles for descriptions.
+.p-strip p a {
```

Comment:
This may be something to address later and not in this PR, but we should not mess up with default typography styles unnecessarily. 

Vanilla has implemented typography based on our design standards and accessibility best practices. Any override like this may be breaking those.

On top of that the selector of `.p-strip p a` is very generic, affecting all the links in strips on the whole website which is likely not intended.

I'd suggest removing those overrides, or make them more specific to video descriptions.